### ISSUE TITLE
Coordinate w/ Devops the creation of an "instance group" to host multiple Cromwells

### ISSUE THREAD

**Issue by danbills** [broadinstitute/cromwell#4798]
## Today

Cromwell workers are deployed as a Terraform `instance,` most likely w/ a `count` of 1.*

### # of instances

The `instance` stanza should reflect a minimum # of instances = `1`, max=`4`.  The 1 represents the normal case where we want 1 Cromwell running, and only scales up when necessary.  

### Autoscaling

Should be set to `CPU Usage` where target == `50%`

Add an additional `group` stanza that creates a GCP instance group and refers to the Cromwell workers.*

*Raph Luckom made this presumption, the author does not have access to verify.

**Comment by aednichols** [comment]
Filed https://broadinstitute.atlassian.net/browse/DSPTO-710

After a discussion with our devops representatives in channel deploy-horicromtal, I cut the autoscaling part for now because managed instance groups don't work in our env for reasons I don't fully understand (and can certainly be revisited later!)

**Comment by aednichols** [comment]
tl;dr is if we can spawn an instance from an instance template, then we can use autoscaling

**Comment by gemmalam** [comment]
Issue moved to Jira
https://broadworkbench.atlassian.net/browse/BA-4798
